packages/kas/src/version.ts

```diff
@@ -2,7 +2,11 @@
 // version number during the release build.
 // In dev, you'll never see the version number.
 
-import {addLibraryVersionToPerseusDebug} from "@khanacademy/perseus-core";
+// ESLint's import/no-relative-packages doesn't understand symlinks! If this
+// lint suppression is removed, it autofixes the import path to an invalid
+// package+path entry.
+// eslint-disable-next-line import/no-relative-packages
```

Comment:
I was wondering if maybe these comments should explain why this is being done since I can imagine something maybe trying to fix this down the road and moving it back to perseus-core. Maybe this PR is enough to prevent that though  XD

Reply:
I edited these comments to hopefully be clearer. :)

Reply:
Thank you! Looks great :D